drivers/network/meson/Makefile

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright 2023, UNSW
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+
+ifeq ($(strip $(MICROKIT_SDK)),)
+$(error MICROKIT_SDK must be specified)
+endif
+
+MICROKIT_CONFIG ?= debug
+MICROKIT_BOARD := odroidc4
+CPU := cortex-a55
+
+TOOLCHAIN := clang
+CC := clang
+LD := ld.lld
+TARGET := aarch64-none-elf
+MICROKIT_TOOL ?= $(MICROKIT_SDK)/bin/microkit
+
+LIBSHAREDRINGBUFFER=../../../network/libethsharedringbuffer
+
+BOARD_DIR := $(MICROKIT_SDK)/board/$(MICROKIT_BOARD)/$(MICROKIT_CONFIG)
```

Comment:
Could do the same thing here as in the Makefile for the timer, but fine if we leave it for later. Maybe leave a TODO at least.